src/Functions/custom/Get-AzFunctionAppAvailableLocation.ps1

```diff
@@ -0,0 +1,52 @@
+function Get-AzFunctionAppAvailableLocation {
+    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.Functions.Models.Api20160301.IGeoRegion])]
```

Comment:
Seels like this should have a cmdletbinding

Reply:
Want to fix for preview, unless there is a reason this is not a cmdlet

Reply:
Good catch. Fixed.